tests/test_annoy_console.py

```diff
@@ -0,0 +1,23 @@
+"""module contains tests for the Slog class, specifically the annoying method, which is responsible for slog annoying"""
+
+from slogpy.slog import Slog as slog  # noqa: N813,
+
+
+def test_annoy_unique_message(capsys, msg="This is a annoy test"):  # noqa: D103
+    """Given: a single line annoy message
+
+    When: passed into slog.annoy
+    Then: method will check for annoy message only once and display it.
```

Comment:
This line is not very clear.
What does method mean here?  Is annoy() the method, or is test_annoy_unique_message() the method?
Assuming this test is what you mean by method, the test certainly checks only once, but it doesn't really display it.
Assuming annoy() is what you mean by method, it doesn't check anything, but it should only output the message once.